charts/generic-service/values.yaml

```diff
@@ -47,9 +47,9 @@ ingress:
   # instructions on advised upgrade process - https://mojdt.slack.com/archives/CH6D099DF/p1657794137979789
   # switching this to true with perform a big bang ingress switch with possible downtime as dns changes
   v1_2_enabled: false
+  v0_47_enabled: true
```

Comment:
default is existing behaviour - so existing ingress only defined, although with weighting of `0` which has no impact if only one ingress.